website/templates/util/render_keys.mako

```diff
@@ -1,3 +1,4 @@
+<!-- TODO: File is unused; candidate for deletion? -->
```

Comment:
Thought so. :) I wouldn't mind a follow-on cleanup PR, but wanted to limit the scope of this one and not also go deleting things. Too many balls in the air at once. :)